src/pudl_archiver/zenodo/api_client.py

```diff
@@ -25,30 +23,6 @@
 logger = logging.getLogger(f"catalystcoop.{__name__}")
 
 
-class ZenodoClientException(Exception):
```

Comment:
Farewell, sweet prince; and flocks of angels sing thee to thy rest.

Reply:
You served us well